runtime/Cargo.toml

```diff
@@ -95,7 +114,10 @@ std = [
 	"polkadex-fungible-assets/std",
 	"polkadex-ocex/std",
 	"pallet-substratee-registry/std",
-	"pallet-proxy/std"
+	"pallet-proxy/std",
+	"chainbridge/std",
+	'erc721/std',
+    'example/std',
```

Comment:
formatting

Reply:
@frankli-dev @mambisi @ksr30 Please use rust-fmt before upload. It solves these issues

Thanks @ayushmishra2005

Reply:
done. (I've installed TOML formatter to my vscode)